samples/DLLMethodCaller/DLLMethodCallerEA.mq4

```diff
@@ -41,36 +41,59 @@
 	int nquotes_get_property_array_size(string name);
 	int nquotes_get_property_adouble(string name, double& value[]);
 #import
+//extern int MoneyAmount = 12345;
+//extern datetime MyBirthday;
+//extern bool AmMarried;
 
 int init()
 {
+   EventSetTimer(5);
 	nquotes_setup("DLLMethodCaller.DLLMethodCallerEA", "DLLMethodCaller");
-	
 	return (nquotes_init());
 }
 
 int start()
 {
-	return (nquotes_start());
+   GiveMoney(10);
+   nquotes_set_property_datetime ("MyBirthday",TimeCurrent() - 1000000000);
+   nquotes_set_property_string ("ActionName","PredictFuture");
+	Comment("FuturePrediction is: " + PredictFuture());
+	GiveMoney(100);
+	nquotes_set_property_datetime ("MyBirthday",TimeCurrent() - 1000000000);
+	nquotes_set_property_string ("ActionName","PredictFuture");
+	Comment("FuturePrediction is: " + PredictFuture());
```

Comment:
the code looks good now, but you don't need nquotes_set_property_xxx calls here anymore